src/main/java/ch/njol/skript/conditions/CondIsEnchanted.java

```diff
@@ -33,50 +31,62 @@
 import ch.njol.skript.lang.SkriptParser.ParseResult;
 import ch.njol.skript.util.EnchantmentType;
 import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
 
-/**
- * @author Peter Güttinger
- */
 @Name("Is Enchanted")
 @Description("Checks whether an item is enchanted.")
 @Examples({"tool of the player is enchanted with efficiency 2",
-		"helm, chestplate, leggings or boots are enchanted"})
-@Since("1.4.6")
+		"helm, chestplate, leggings or boots are enchanted",
+		"",
+		"# For enchanted books",
+		"player's tool is stored enchanted",
+		"event-item is enchanted with stored power 2"})
```

Comment:
```suggestion
		"event-item is enchanted with stored power 2"
})
```
same with the top bracket